### ISSUE TITLE
Choose other inputs for astra projector setting 

### ISSUE THREAD

**Issue by epapoutsellis** [vais-ral/CCPi-astra#23]
https://github.com/vais-ral/CCPi-astra/blob/master/Wrappers/Python/ccpi/astra/processors/AstraForwardProjector.py#L42

line, linear etc

**Comment by paskino** [comment]
@epapoutsellis what does this mean?

**Comment by epapoutsellis** [comment]
In parallel geometry , astra provides 3 different approximations see [here](https://www.astra-toolbox.com/docs/proj2d.html).
As a default we have `line`. Maybe we can add the other two. @jakobsj what do you think?

**Comment by jakobsj** [comment]
Yes that would be nice. Note that those three are all on the cpu. On the
gpu, there's only one option. That can be chosen in a similar way to the
others but specifying 'cuda' instead of 'line,'strip', or 'linear'.
